arb_os/constants.json

```diff
@@ -107,6 +107,7 @@
     "EVMCallType_staticcall": 3,
     "EVMNonZeroBalanceCallStipend": 2300,
     "EVMWriteL1GasCost": 5000,
+    "EscrowStoreID_txGas": 0,
```

Comment:
It looks like EscrowStoreID_txGas is set to two different values in this file

Reply:
Removed one